user.js

```diff
@@ -1,3 +1,4 @@
+//
```

Comment:
This line must be set in `mozilla.cfg`.
I included it i `user.js` to simplify the installation for the users.

Reply:
Any idea why this is required? Why doesn't it work properly without it?

Reply:
Apparently, Firefox skips the first line containing `//` on purpose, to prevent loading malicious, arbitrary JavaScript code.    

Source: [Archlinux](http://kb.mozillazine.org/Locking_preferences)
>  The parser ignores the first line of this file under the assumption that it was made unparseable on purpose "to protect the privacy of the JavaScript preferences file from loading by the browser" (code imported by [bug 89137](https://bugzilla.mozilla.org/attachment.cgi?id=61802&action=diff#mozilla/extensions/pref/autoconfig/src/nsJSConfigTriggers.cpp_sec2)). 

Source: [Mozilla's bugzilla bug 89137](https://bugzilla.mozilla.org/attachment.cgi?id=61802&action=diff#mozilla/extensions/pref/autoconfig/src/nsJSConfigTriggers.cpp_sec2)
> In order to protect the privacy of the JavaScript preferences file from loading by the browser, we make the first line unparseable by JavaScript. We must skip that line here before executing the JavaScript code.